modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -36,244 +39,320 @@ from Canvas Kit v6 to v7:
-requests to increase this, we have upped it by 25% to `200px`.
+- `Box` from `@workday/canvas-kit-labs/common` to `@workday/canvas-kit-react/layout`
+- `Flex` from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
+- `Stack` from `@workday/canvas-kit-labs/layout` to `@workday/canvas-kit-react/layout`
 
-We've also added a prop (`maxWidth`) to configure this if you need to. While this increases
-flexibility, please keep in mind that status text should be short, direct, and preferably a single
-word.
+🤖 The codemod will update imports for these promoted components.
 
 ```tsx
-<StatusIndicator label="Slightly Longer Status" type={StatusIndicator.Type.Gray} maxWidth={250} />
+// v6
+import {Box} from '@workday/canvas-kit-labs-react/common';
+
+// v7
+import {Box} from '@workday/canvas-kit-react/layout';
 ```
 
-## Buttons
+Note that the APIs for these promoted components remain unchanged.
+
+---
 
-To consolidate button APIs, we removed `IconButton` in favor of `PrimaryButton`, `SecondaryButton`
-and `TertiaryButton`.
+### Action Bar
 
-### Breaking Changes
+We've updated `ActionBar` to use the
+[createComponent](/getting-started/for-developers/resources/creating-compound-components/#disclosurecontent-component)
+utility from our
+[common](https://github.com/Workday/canvas-kit/blob/ff77c5bd83e41c3ab2b9c55e41a8b7c1fde33a1b/modules/react/common/lib/utils/components.ts#L167)
+module; it now supports [ref forwarding](https://reactjs.org/docs/forwarding-refs.html) and using
+the `as` prop to change the rendered element.
 
-- Removed the `toggled` prop when migrating from `IconButton`
-- Changed the values of `IconPosition: left | right` - > `IconPosition: start | end`
-- Remove `dataLabel` prop from `PrimaryButton` and `SecondaryButton`
+We've removed the `fixed` prop from `ActionBar`. It now uses the `position` prop (defaults to
+`fixed`) to set the container position.
 
-🤖 The codemod will handle all these changes automatically:
+🤖 The codemod will remove uses of the `fixed` prop from `ActionBar`.
 
-- `iconPosition='left'` becomes `iconPosition='start'`
-- `iconPosition='right'` becomes `iconPosition='end'`
-- `<IconButton icon={plusIcon} />` becomes `<TertiaryButton icon={plusIcon} />`
-- `<IconButton variant='circleFilled' icon={plusIcon} />` becomes
-  `<SecondaryButton icon={plusIcon} />`
-- `<IconButton variant='inverse' icon={plusIcon} />` becomes
-  `<TertiaryButton variant='inverse' icon={plusIcon} />`
-- `<IconButton variant='inverseFilled' icon={plusIcon} />` becomes
-  `<SecondaryButton variant='inverseFilled' icon={plusIcon} />`
+---
 
-#### Manual Changes
+### Banner
 
-The following changes will not be automatic updated and will require a manual update:
+Banner is now a [compound component](/getting-started/for-developers/resources/compound-components/)
+composed of `Banner.Icon`, `Banner.Label`, and `Banner.ActionText`. This allows direct access to the
+icon, label, and text elements.
```

Comment:
I could use an extra pair of eyes on the Banner section. I had to write it from scratch since it was never added the to the v7 migration guide (the guide didn't exist when the [PR](https://github.com/Workday/canvas-kit/pull/1359/files) was merged in). cc: @vibdev